### ISSUE TITLE
Changing and/or removing the welcome message?

### ISSUE THREAD

**Issue by koesherbacon** [zsh-users/fizsh#11]
I'm having a little difficulty changing and/or removing the welcome message:

```
welcome to fizsh, the friendly interactive zshell
type man fizsh for instructions on how to use fizsh
```

Could you tell me how to go about changing it?  I have already tried changing and removing it from `~/.fizsh/fizsh` to no avail.

In case it makes any difference, here's what I would like it to say as a reminder to myself about my aliases:

```
################################################
#
# Print the welcome message
#
function _fizsh-welcome-message () {
  echo 'Fizsh - The Friendly Interactive Zsh-Shell'
  echo -e 'type \033[03;32mleaf $HOME/.fizsh/.fizshrch\033[00m or simply \033[03;32mevz\033[00m to add/remove/update aliases'
}

################################################
```

Thanks a lot!

~ Ev

**Comment by guidovansteen** [comment]
Hi Ev, 

Thank you for your comment. 

fizsh is usually called from the directory /usr/bin/ (or /usr/local/bin/), which is probably on your path as well. So you could change the function _fizsh-welcome-message in the script /usr/bin/fizsh. 

If you want to change ~/.fizsh/fizsh, you could call fizsh as ~/.fizsh/fizsh (instead of /usr/bin/fizsh). 

You could define an alias to make sure that an unqualified "fizsh" resolves to "~/.fizsh/fizsh". E.g. alias fizsh=~/.fizsh/fizsh

Best wishes, 

Guido

**Comment by katafrakt** [comment]
@guidovansteen Would it be possible to completely remove welcome message, eg. with some option switch? It's kind of annoying when I try to use it as my main shell in Terminator and every time I open new pane it shows up (and returns status `1` to make things more fun :wink:).

**Comment by guidovansteen** [comment]
Thank you for the idea! I never cared about suppressing the welcome message in fish. That is why I never created a way to change the welcome message in fizsh either. I will provide an option in the next release. In the next release I will correct this stupid return-status bug as well. 

Meanwhile you can of course edit the source code like @koesherbacon suggested.

**Comment by guidovansteen** [comment]
I just released version 1.0.9. Starting from this version fizsh will only provide a welcome message if called as a login shell. So `fizsh -l` will display the login message, whereas `fizsh` (without -l or --login) will not display it.
